hardhat.config.ts

```diff
@@ -25,7 +25,7 @@ const getAccounts = (network: string) => {
 const config: HardhatUserConfig = {
   defaultNetwork: "hardhat",
   solidity: {
-    compilers: [{ version: "0.8.4", settings: {} }],
+    compilers: [{ version: "0.8.6", settings: {} }],
```

Comment:
Do we intend to go out in September using the most up-to-date version of Solidity?

Reply:
We will try